cea/demand/demand_main.py

```diff
@@ -113,7 +113,7 @@ def demand_calculation(locator, weather_path, gv):
 def write_totals_csv(building_properties, locator, gv):
     """read in the temporary results files and append them to the Totals.csv file."""
     counter = 0
-    for name in building_properties.list_building_names():
+    for name in building_properties.list_building_names()[:1]:
```

Comment:
woah! this is probably a very bad idea!! (it could end up in production, like right now, it nearly went to production, if we had merged....)